src/styles/tabs.js

```diff
@@ -57,7 +53,10 @@ export const materialTopTabNavigatorConfig: Props => MaterialTopTabNavigatorConf
       indicatorStyle: {
         backgroundColor: BRAND_COLOR,
       },
-      upperCaseLabel: false,
+      // TODO: `upperCaseLabel` vanished in react-navigation v5; we
+      // used to use `false` for this, but it appears that the
+      // effective default value is `true`:
+      // https://github.com/react-navigation/react-navigation/issues/7952
```

Comment:
Ah OK, I've figured out why this doesn't have a visible effect.

It _would_ have had a visible effect if we had used a string instead of a React element for `tabBarLabel`, or if we hadn't specified `tabBarLabel` and defined [`title`](https://reactnavigation.org/docs/material-top-tab-navigator/#title).

By passing a React element, we get to say exactly how the tab bar label should look, and that includes case.

Reply:
That might mean I can delete this TODO, but I have a feeling we'll run up against it again if we ever stop using our strategy here, or we make a material top tab nav somewhere else that doesn't use that strategy.

Reply:
Hmm interesting!

Yeah, I think that means the TODO still belongs here. It'd be good to also add a comment with that information you just learned about when it would start affecting things.

Reply:
Oh, hmm. With these findings in mind, I've just found a place where I'd expect to see a change: The bottom tab nav in `MainTabs` shows tab-bar labels, which we define with strings like "Home", "Streams", etc., if `!!Platform.isPad`. On an iPad simulator, I do indeed see those labels—but I don't see them in all caps.

I suspect the all-caps quirk applies to the "material top tab" navigator, but doesn't apply to the "bottom" tab navigator. I wouldn't be totally surprised to see differences spring up between them; the React Nav v5 upgrade took us from depending on `react-navigation-tabs` to `@react-navigation/material-top-tabs` and `@react-navigation/bottom-tabs`.

Reply:
Yeah, I think that's an intentional design choice.

Or more specifically, it's a faithful implementation (at least on that point) of Material Design, which those top tabs are advertised as being for. Material calls those top tabs simply "tabs", and they are in caps:
https://material.io/components/tabs
![image](https://user-images.githubusercontent.com/28173/104525368-d91d9180-55b4-11eb-85d3-500c4baddb1d.png)

For our bottom tabs, Material's term is "bottom navigation":
https://material.io/components/bottom-navigation
and the titles are not in caps.

Meanwhile, Apple says "tab bar" for something very much like our bottom tabs, or Material's bottom navigation:
https://developer.apple.com/design/human-interface-guidelines/ios/bars/tab-bars/
and those also aren't in caps. I'm not sure there's any equivalent to top tabs, or Material tabs, in Apple's iOS guidelines.

Anyway, I think that means that we shouldn't expect the upper-case thing to just randomly change on us in the future due to something intended as a bugfix.